src/command_modules/azure-cli-cognitiveservices/tests/test_cognitiveservices_command.py

```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=method-hidden
+#pylint: disable=line-too-long
+#pylint: disable=bad-continuation
+#from _future_ import print_function
+
+import os
+import time
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, JMESPathCheck, ResourceGroupPreparer
+from azure.cli.core.util import CLIError
+
+class CognitiveServicesTests(ScenarioTest):
+    @ResourceGroupPreparer()
+        
+    def test_cognitiveservices(self, resource_group):
```

Comment:
Please use a better and more elaborate test name. There will be more test cases eventually.